content/linked-terms.js

```diff
@@ -0,0 +1,6 @@
+module.exports = [
+  {
+    matches: 'dvc.yaml',
+    url: '/doc/user-guide/dvc-files-and-directories#dvcyaml-file'
+  }
```

Comment:
```suggestion
  }
  {
    matches: 'dvc.yaml',
    url: '/doc/user-guide/dvc-files-and-directories#dvcyaml-file'
  }
  {
    matches: 'dvc.lock',
    url: '/doc/user-guide/dvc-files-and-directories#dvclock-file'
  }
  {
    matches: ['`.dvc` file', '`.dvc` file'],
    url: '/doc/user-guide/dvc-files-and-directories#dvclock-file'
  }
```
The last one probably won't work like this though. It's tricky because the back quotes don't wrap the full text...

Reply:
> The last one probably won't work like this

If that's too difficult to implement or problematic for some reason, we can just link `.dvc` I guess. That should be fine and it's easier, so that last one would be:

```js
  {
    matches: '`.dvc`,
    url: '/doc/user-guide/dvc-files-and-directories#dvclock-file'
  }
```

Reply:
> We just have to be careful to always use `.dvc/` when referring to the directory.